src/sonic-frr-mgmt-framework/frrcfgd/frrcfgd.py

```diff
@@ -2258,7 +2269,7 @@ def __init__(self):
         syslog.syslog(syslog.LOG_DEBUG, 'Init Cached DB data')
         for key, entry in self.table_data_cache.items():
             syslog.syslog(syslog.LOG_DEBUG, '  %-20s : %s' % (key, entry))
-        if self.config_mode == "unified":
+        if self.config_mode == "unified" or self.config_mode == "separated":
```

Comment:
Is "unified" config_mode not working for you use-case? seperated config_mode supposed to generate the individual daemon config files during boot-up as per design in docker_init.sh, we should avoid duplicate config generation.

Reply:
That was my original question how "fpm address 127.0.0.1" is applied in "unified" mode?

Reply:
I believe, we had config loading time issue when we push the configs one by one during boot-up incase of scaled configs instead of generating the static config file per FRR daemon, if you have any perf data, if would be useful. If you think, this new change gives better performance and avoids additional jinja file creation, I would like to have a new config_mode or a field (in DEVICE_METADATA) defined for this purpose to avoid overloading of the seperated mode design today.

Reply:
Ok. In unified mode, how does "fpm address 127.0.0.1" get applied? I could use "unified" mode if I could apply this config.

Reply:
ok, I'll help you for the above one.

Reply:
You can add the fpm address command in the following j2 file to work for unified mode, please check.
https://github.com/sonic-net/sonic-buildimage/blob/master/src/sonic-frr-mgmt-framework/templates/frr/frr.conf.j2

Reply:
Changed to unified mode and set fpm address in frr.conf.